src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/LegalRepresentation.java

```diff
@@ -23,25 +25,33 @@
 @Getter
 @Setter
 @NoArgsConstructor
-public class TicketCount {
-
+public class LegalRepresentation {
+	
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
-	@Id
-	@GeneratedValue
+    @Id
+    @GeneratedValue
     private UUID id;
-
+	
 	@Column
-	private String offenceDeclaration;
-
+	private String lawFirmName;
+	
 	@Column
-	private boolean timeToPayRequest;
-
+	private String lawyerName;
+	
+	@Column
+	private String lawyerSurname;
+	
 	@Column
-	private boolean fineReductionRequest;
+	@Email(regexp = ".+@.+\\..+")
```

Comment:
We can improve latter on, but I think the typical email regex is more like
"^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$"
https://regexr.com/3e48o